packages/plugin-core/src/survey.ts

```diff
@@ -772,15 +812,17 @@ export class SurveyUtils {
           const onboardingSurvey = InactiveUserOnboardingSurvey.create();
```

Comment:
slight update to this logic. instead of doing this in vscode, can we navigate the user to the following airtable form? https://airtable.com/shry4eLgvVE6WR0Or?prefill_SurveyName=InactiveFeedback